hazelcast/include/hazelcast/client/connection/ClientConnectionManagerImpl.h

```diff
@@ -126,6 +126,8 @@ namespace hazelcast {
 
                 void notify_backup(int64_t call_id);
 
+                std::unique_ptr<load_balancer> get_load_balancer();
```

Comment:
return reference?

Reply:
This one was actually left over, it has no definition. Removed it.